pkgs/development/coq-modules/extructures/default.nix

```diff
@@ -9,13 +9,15 @@
 
   inherit version;
   defaultVersion = with lib.versions; lib.switch [coq.coq-version ssreflect.version] [
+    { cases = [(range "8.17" "8.18") (isGe "2.0.0")  ]; out = "0.4.0"; }
     { cases = [(range "8.11" "8.18") (isGe "1.12.0") ]; out = "0.3.1"; }
     { cases = [(range "8.11" "8.14") (isLe "1.12.0") ]; out = "0.3.0"; }
     { cases = [(range "8.10" "8.12") (isLe "1.12.0") ]; out = "0.2.2"; }
   ] null;
 
   releaseRev = v: "v${v}";
 
+  release."0.4.0".sha256 = "sha256:1jbvawwmmczf9algdq81iqnrlmwwl8cxhwyrsi1kq3fmpvanx2f1";
```

Comment:
```suggestion
  release."0.4.0".sha256 = "sha256-hItFO2XY2LTPSofPTKt3AfOEfiLliaYdzUXgDv4ea9Y=";
```

Reply:
This is in base 64, right? For future reference, what is the easiest way to output the hash in this format?

Reply:
This is not a matter of format: the value is different.

Reply:
@vbgl I am not sure I understand the issue. When I run

```
$ nix-prefetch-url --type sha256 https://github.com/arthuraa/extructures/archive/refs/tags/v0.4.0.tar.gz
```

I get the following output

```
path is '/nix/store/ka3l2rvylq06sy2fps1g25yb7w3lfgc7-v0.4.0.tar.gz'
1jbvawwmmczf9algdq81iqnrlmwwl8cxhwyrsi1kq3fmpvanx2f1
```

I took this value to fill in the `sha256` field in the format, but you seem to suggest that I should have computed the hash using some other command. How did you do it?